### PR TITLE
Improve dev/releases/README.md

### PR DESCRIPTION
**Pull request by fingolfin**
Add several reminders:
- one should check the outcome of the "Wrap releases" CI job
- announce the release on the GAP mailing lists
- enable the "latest release" flag and disable the "prerelease" flag

### CODE REVIEW

**Review by ruthhoffmann — COMMENTED** [review]
Just checking, are we expecting the version number always to be in the format `X.Y.Z`?
I.e. are we expecting in major releases `Z` = 0?

**Comment by fingolfin** [comment]
Indeed. So 4.12.0, not 4.12

File: dev/releases/README.md [hunk 1]
```diff
@@ -101,8 +104,13 @@ Before starting the release process, the scripts have the following dependencies
 11. Check that <https://docs.gap-system.org> is functioning as expected.
 
 
-## Post-release hints
+## Post-release instructions
+
+Once all the above has been completed, do the following:
 
+- Write a release announcement to the GAP Forum and GAP mailing list
+- Go to the new released under <https://github.com/gap-system/gap/releases/> and
```
Comment (ruthhoffmann) [inline dev/releases/README.md:112]: released -> releases or release ?

Comment (fingolfin) [reply dev/releases/README.md]: thanks, fixed

**Review by wilfwilson — APPROVED** [review]

**Pull request merged**